.github/workflows/build-publish.yaml

```diff
@@ -0,0 +1,71 @@
+---
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Build and publish the extension
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
does it also require a daily/weekly job to rebuild/update the images ?

Reply:
No build is scheduled only if Containerfile is changed